website/api/use-cases.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: /use-cases
+---
+
+# Use Cases
+
+Verifiable computation is a game changer for the resilience and economics of
+operating the computing infrastructure we all rely on. It creates a number of
+emergent use cases which we are excited to enable. Key among these use cases
+are:
+
+- [zk coprocessors], which enable on-chain applications to reduce gas costs by
+  moving the complex part of their application logic off-chain
+- blockchain infrastructure, including [our work with Optimism][optimism]
```

Comment:
good call-out!  

if you have a suggested revision i'm happy to include an improvement to this on this PR, otherwise let's save it for a future improvement -- I'm just copying/pasting this section from elsewhere on the site.

Reply:
Ya, I dislike the way this section is written. I wrote some suggestions, although I'm still not totally very happy with it https://github.com/risc0/risc0/pull/1484

Reply:
awesome -- thanks victor!